Package.swift

```diff
@@ -9,23 +9,26 @@ let package = Package(
     products: [
         .library(
             name: "ZXingCppWrapper",
-            type: .static,
+            type: .dynamic,
             targets: ["ZXingCppWrapper"])
     ],
```

Comment:
Not really, probably they were missing some core libraries. With the old variant, you had to also add these frameworks to your app (CoreGraphics, CoreImage, CoreVideo). I can't see any benefit here in using a static library, it will increase your overall app size without any benefit at all.

Not quite sure how the original author used it, but there were a few downsides which avoided one from submitting an app to the App Store (umbrella framework) or even to build the app (unsafeFlags).